UPGRADE-7.1.md

```diff
@@ -27,14 +59,31 @@ ExpressionLanguage
  * Deprecate passing `null` as the allowed variable names to `ExpressionLanguage::lint()` and `Parser::lint()`,
    pass the `IGNORE_UNKNOWN_VARIABLES` flag instead to ignore unknown variables during linting
 
+Form
+----
+
+ * Deprecate not configuring the `default_protocol` option of the `UrlType`, it will default to `null` in 8.0 (the current default is `'http'`)
+
 FrameworkBundle
 ---------------
 
  * Mark classes `ConfigBuilderCacheWarmer`, `Router`, `SerializerCacheWarmer`, `TranslationsCacheWarmer`, `Translator` and `ValidatorCacheWarmer` as `final`
+ * Deprecate the `router.cache_dir` config option, the Router will always use the `kernel.build_dir` parameter
+
+Intl
+----
+
+ * Extracted `EmojiTransliterator` to a separate component, the new FQCN is `Symfony\Component\Emoji\EmojiTransliterator`
```

Comment:
```suggestion
 * Extracted `EmojiTransliterator` to a separate `symfony/emoji` component, the new FQCN is `Symfony\Component\Emoji\EmojiTransliterator`
```

Reply:
And I think this should have the `[BC break]` prefix as an application not installing the new package would not get an exception when trying to use the old EmojiTransliterator class instead of having a working transliterator.

Reply:
There is an exception if you uses the old one no ? 

https://github.com/symfony/symfony/blob/7.1/src/Symfony/Component/Intl/Transliterator/EmojiTransliterator.php

Reply:
Thanks, I've applied both @stof's suggestions.

While we show a nice error, I agree that this is a BC break as we still create an error instead of handling it gracefully (having old behavior without any change).